ShinyExpenseTracker/src/ca/ualberta/cs/shinyexpensetracker/ExpenseItemDetailView.java

```diff
@@ -0,0 +1,116 @@
+/**
+ * Activity that lets the user view 
```

Comment:
I believe the Javadoc needs to be right above the class declaration, not at the top of the file, so that Javadoc picks it up properly.